artifactory/cli.go

```diff
@@ -484,6 +498,15 @@ func GetCommands() []cli.Command {
 	}
 }
 
+func getGlobalConfigFlag() []cli.Flag {
+	return []cli.Flag{
+		cli.BoolFlag{
+			Name:  "global",
+			Usage: "[Default: false] Saves the config file to JFROG_CLI_HOME_DIR.` `",
```

Comment:
Saves the config file to JFROG_CLI_HOME_DIR. --> Set to true, if you'd like to configuration to be global (for all projects). Specific projects can override the global configuration.

Reply:
Done. Should it be:  Set to true, if you'd like **the** configuration to be global (for all projects). Specific projects can override the global configuration. ?